packages/app/src/cli/models/app/loader.ts

```diff
@@ -126,15 +127,6 @@ export async function parseConfigurationFile<TSchema extends zod.ZodType>(
   return {...configuration, path: filepath}
 }
 
-export function parseHumanReadableError(issues: Pick<zod.ZodIssueBase, 'path' | 'message'>[]) {
```

Comment:
We still need to export the `parseHumanReadableError` function right? Otherwise it wouldn't be available to use from the loader.

Reply:
Good question. I think it's fine considering no other class is using the `parseHumanReadableError` function, and it's now exported from `error-parsing.ts` file [here](https://github.com/Shopify/cli/blob/ff432d01c9d93c9b27318835d49423212d2f3570/packages/app/src/cli/models/app/error-parsing.ts#L65). In `loader.ts`, it is [imported](https://github.com/Shopify/cli/blob/ff432d01c9d93c9b27318835d49423212d2f3570/packages/app/src/cli/models/app/loader.ts#L22). If I missed anything, please point it out TY! 😺